### PR TITLE
actions/setup-elixir now lives under erlef GH Org

### PR DESCRIPTION
**Pull request by iautom8things**
[actions/setup-elixir](https://github.com/actions/setup-elixir) is no longer maintained and now suggests using [erlef/setup-elixir](github.com/erlef/setup-elixir)

This should be updated to help new Github Actions users to use the supported action.

### CODE REVIEW

**Review by andymckay — APPROVED** [review]
Thanks @iautom8things. This looks good to me, cc @jclem?

**Review by andymckay — CHANGES_REQUESTED** [review]
Actually... we'll need to pin this to a 40 char SHA instead of a version because its no longer in the actions repo.

File: ci/elixir.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Elixir
-      uses: actions/setup-elixir@v1
+      uses: erlef/setup-elixir@v1
```
Comment (andymckay) [inline ci/elixir.yml:18]:
```suggestion
      uses: erlef/setup-elixir@885971a72ed1f9240973bd92ab57af8c1aa68f24

```

**Review by jclem — APPROVED** [review]
This looks good to me! Thanks for the PR @iautom8things.

**Review by andymckay — APPROVED** [review]

**Comment by iautom8things** [comment]
Thank you @jclem and @andymckay!  🙏 ❤️ 💙 💚

**Pull request merged**